CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
   If you are using a custom storage adapter or consume `ReadPage` func in your code, you will need to update the return type and/or handling of the `ReadPage` function.
 * `ErrMismatchObjectType` error type removed from `openfga` package [#2064](https://github.com/openfga/openfga/pull/2064) as storage is not validating this anymore. 
   Validation moved to `ReadChangesQuery` implementation.
+* Removed flags with the `OPENFGA_DISPATCH_THROTTLING_*` name.
```

Comment:
**Suggestion:** To abide by KACL, this should exist under a `## Removed` heading. See: https://keepachangelog.com/en/1.0.0/